src/unleash-proxy.ts

```diff
@@ -80,15 +168,15 @@ export default class UnleashProxy {
             const context = createContext(query);
             const toggles = this.client.getEnabledToggles(context);
             res.set('Cache-control', 'public, max-age=2');
-            res.send({ toggles });
+            res.send({ toggs: toggles });
```

Comment:
:eyes:

Reply:
🤔

Reply:
I thought I fixed this. Interestingly enough, it seems to catch _some_ type errors, but not all of them. I'll see what more I can find out.